packages/slate/src/models/history.js

```diff
@@ -64,8 +82,8 @@ class History extends Record(DEFAULTS) {
     const { redos = [], undos = [] } = object
 
     const history = new History({
-      redos: new Stack(redos),
-      undos: new Stack(undos),
+      redos: new Stack(redos.map(this.createList)),
```

Comment:
should this just be `this.createList(redos)` / `this.createList(undos)`?

Reply:
@jtadmor nope, the stack is fine being generated from any iterable, but this class expects the stack to contain lists so we have to map all the things that will be in the stack to lists so that we can use `.last`

Reply:
ah i see, thanks.

Reply:
its needed for when the stored object has been serialized and `List` gets converted to an `Array` and `History` previously did not re-box it